app/models/episode.rb

```diff
@@ -59,6 +65,10 @@ def enclosure
     enclosures.complete.first
   end
 
+  def image
+    images.complete.first
```

Comment:
this is not the default image, it is the current image for the episode.
There will be multiple images in `images` when we have an updated image, and there is a current image that is there, and then a new image is being processed and will replace it.